js/hview.js

```diff
@@ -828,6 +853,25 @@ $(function () {
         $("#vlist_content").showError(msg ? msg : "Error loading view hierarchy")
     }
 
+    function updateNodeRelationships(node) {
+        const el = document.getElementById(node.treeDisplayName)
+        node.disablePreview = el.node.disablePreview
```

Comment:
can you move this to the diff logic instead of a second iteration?

Reply:
Yes, I can.

Reply:
Update: It was unnecessary. Instead, the DOM is no longer the source of truth, but rather the rootNodes[] information is. The DOM's node references are being updated with every compare, since it is being traversed anyways when comparing 2 frames to each other for differences.